src/restruct_poc/system/calculate_Ouu.hpp

```diff
@@ -9,6 +10,9 @@
 namespace openturbine {
 
 struct CalculateOuu {
+    using NoTranspose = KokkosBatched::Trans::NoTranspose;
+    using Default = KokkosBatched::Algo::Gemm::Default;
+    using Gemm = KokkosBatched::SerialGemm<NoTranspose, NoTranspose, Default>;
     View_Nx6x6::const_type qp_Cuu_;
```

Comment:
Would it make sense to define these type aliases in `src/restruct_poc/types.hpp` as opposed to defining them in several places of system files?

Reply:
I like keeping aliases like this local, in case we want to do specific tweaking to the algorithm choice/configuration.  It also saves common names like these for if we want to use them differently in other contexts